### DISCUSSION TITLE
Issue with the API

### DISCUSSION THREAD

**Discussion by bastienlaval** [plausible/analytics#829] [Anything Else]
Hello,

I’ve got an issue with the API and can’t manage to understand why…

On one side, I make a request to the ‘aggregate’ endpoint, and get exactly the same figures than when I log in the plausible dashboard.
Here’s my endpoint :
https://plausible.io/api/v1/stats/aggregate?site_id=my-site.com&period=12mo&metrics=visitors,pageviews&filters=event:page==/my-page

And on the other site, I make a request to the ‘breakdown’ endpoint.
I get a result, but the total I get in inferior to the number I get in the previous request. And it’s also different from the number I get inside the plausible dashboard…
Here’s my endpoint (for the country property for instance, but I got the same issue with device property) : 
https://plausible.io/api/v1/stats/breakdown?site_id=my-site.com&period=12mo&property=visit:country&metrics=visitors&filters=event:page==/my-page

Does anyone can help me trying to understand what I’m doing wrong?

Thanks a lot in advance!
Bastien

**Comment by ukutaht** [comment]
Thanks for letting me know about this! I'll be taking a look at what's wrong. I doubt you're doing anything wrong, the backend is probably not querying the database correctly.

**Comment by bastienlaval** [comment]
Here are a few screenshots :
<img width="1337" alt="Capture d’écran 2021-03-12 à 09 14 10" src="https://user-images.githubusercontent.com/21161975/110912924-cc978a00-8314-11eb-935a-651f29ae3439.png">
<img width="1126" alt="Capture d’écran 2021-03-12 à 09 14 31" src="https://user-images.githubusercontent.com/21161975/110912939-cef9e400-8314-11eb-949b-23b56d448ad5.png">
<img width="574" alt="Capture d’écran 2021-03-12 à 09 14 38" src="https://user-images.githubusercontent.com/21161975/110912945-d02b1100-8314-11eb-80b6-a9635a845d14.png">

**Reply by ukutaht** [reply]
Thank you that's helpful. Looks like the page filter is acting funny.  If you remove the page filter, do the numbers match?

**Reply by bastienlaval** [reply]
Hey! There's still a difference if I remove the page filter :
<img width="568" alt="Capture d’écran 2021-03-12 à 09 41 57" src="https://user-images.githubusercontent.com/21161975/110914979-62ccaf80-8317-11eb-91da-d8d00edf2e84.png">
<img width="361" alt="Capture d’écran 2021-03-12 à 09 42 08" src="https://user-images.githubusercontent.com/21161975/110914984-63fddc80-8317-11eb-9342-1915754f805c.png">

**Reply by ukutaht** [reply]
I've fixed it locally, will be live once I merge https://github.com/plausible/analytics/pull/826

This was a pretty subtle bug that slipped by testing. Very grateful for reporting this ❤️

**Reply by bastienlaval** [reply]
ok cool ! thanks a lot !!!
